deploy/charts/istio-csr/values.yaml

```diff
@@ -205,6 +205,8 @@ app:
       # The type of private key to generate for the serving certificate. Only RSA (default) and ECDSA are supported.
       # NB: This variable is named incorrectly; it controls private key algorithm, not signature algorithm.
       signatureAlgorithm: "RSA"
+    # A list of service accounts that are allowed to use node authentication for CSRs, eg. istio-system/ztunnel
+    caTrustedNodeAccounts: ""
```

Comment:
should we default this to `istio-system/ztunnel`?

Reply:
in which situation do we want to change this value?

Reply:
Users may run ztunnel under a different service account. For example https://istio.io/latest/docs/ambient/install/platform-prerequisites/#google-kubernetes-engine-gke